web-jehlomat/src/screens/EditaceUzivatele/EditaceUzivatele.tsx

```diff
@@ -97,6 +102,30 @@ const EditaceUzivatele: React.FC = () => {
         [history, userId],
     );
 
+    const removeUserFromOrganization = useCallback(async () => {
+        try {
+            if (!user?.id) {
+                return;
+            }
+            const confirmResult = await confirmationModal?.show({
+                title: `Odstranit uživatele ${user?.username} z organizace ${organization?.name}?`,
+                confirmText: 'Odstranit?',
+                cancelText: 'Zrušit',
+            });
+            if (!confirmResult || confirmResult === 'cancel') {
+                return;
+            }
+            const response: AxiosResponse<any> = await API.delete(apiURL.deleteUserFromOrganization(user.id));
+            if (isStatusGeneralSuccess(response.status)) {
+                history.push(LINKS.USER);
+            } else {
+                throw new Error();
+            }
+        } catch (error: any) {
+            history.push(LINKS.ERROR);
+        }
+    }, [confirmationModal, user, organization?.name, history]);
```

Comment:
Correct me, I may be wrong, but I think the `useCallback` wouldn't work in this case thanks to the `history`, which is unstable function, so at the end of the day you'll always pass a new instance of function to the `TextButton` component.
Why do I mention here? Not sure, if there's any benefit of using useCallback here - the TextButton component at a first sign seems to be quite simple component, so re-rendering shouldn't be so problematic, hence the question is if the cost of memorize function creation overweights the benefits of stable function.
How it can be solved? From top of my head, I think you would need to wrap the history function into the ref with combination of useState, please Google for more details, or just remove the history from dependency array with proper eslint rule disable comment.
But again, I may be wrong, so better to check it yourself.

Reply:
I tried to google it, but did not found anything that would cause problem having `history` in dependency array.  The reference to `history` object remains same until the url is changed (and it that case the component will not be shown).. but you are right, that probably we do not need `useCallback` in this case, so i will remove it and keep it as normal function